dev-packages/node-integration-tests/suites/tracing/linking/test.ts

```diff
@@ -0,0 +1,166 @@
+import { createRunner } from '../../../utils/runner';
+
+describe('span links', () => {
+  test('should link spans with addLink() in trace context', done => {
+    let span1_traceId: string, span1_spanId: string;
+
+    createRunner(__dirname, 'scenario-addLink.ts')
+      .expect({
+        transaction: event => {
+          expect(event.transaction).toBe('span1');
+
+          // assigning a string to prevent "might be undefined"
+          span1_traceId = event.contexts?.trace?.trace_id || 'non-existent-trace-id';
```

Comment:
I am not sure if just assigning a string here is a great idea in the test. But the likelihood of the traceID actually being `'non-existent-trace-id'` is pretty low.

Reply:
hm, I think I'm going to just cast this to `as string`

Reply:
yeah I think it's okay! We mostly check for traceId validity via expectStingmatching(/[a-f0-9]{32}/ but this isn't the focus in this test. So I think casting is fine.